app/models/task.rb

```diff
@@ -1,2 +1,6 @@
 class Task < ApplicationRecord
+  enum tag: %i[postponed important urgent]
```

Comment:
Please, enable rubocop and follow its offences.

Reply:
Rubocop says nothig about this row.
But i added some comments as it recommends.

Reply:
And what about https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment ?